File: The_Amazing_Juggler/The_Amazing_Juggler/MenuViewController.m
//
//  MenuViewController.m
//  The_Amazing_Juggler
//
//  Created by greyson on 11/5/15.
//  Copyright © 2015 Greyson Wright. All rights reserved.
//

#import "MenuViewController.h"
#import "AppDelegate.h"
#import "MainViewController.h"
#import <iAd/iAd.h>

@interface MenuViewController ()

@end

@implementation MenuViewController

- (void)viewDidLoad {
	[super viewDidLoad];
	
	self.canDisplayBannerAds = YES;
    // Do any additional setup after loading the view from its nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
	
	((AppDelegate *)[UIApplication sharedApplication].delegate).window.rootViewController = [[MainViewController alloc] init];
	
}

@end

File: The_Amazing_Juggler/MenuViewController.swift
//
//  MenuViewController.swift
//  The_Amazing_Juggler
//
//  Created by Greyson Wright on 6/2/16.
//  Copyright © 2016 Greyson Wright. All rights reserved.
//

import UIKit

class MenuViewController: UIViewController {
	@IBOutlet weak var highScoreLabel: UILabel!
	
    override func viewDidLoad() {
        super.viewDidLoad()

		if let highScore = NSUserDefaults.standardUserDefaults().valueForKey("HighScore") {
			highScoreLabel.text = "High Score: \(highScore)"
		} else {
			NSUserDefaults.standardUserDefaults().setInteger(0, forKey: "HighScore")
		}
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
}
